types/externs.d.ts

```diff
@@ -219,44 +220,8 @@ declare global {
       [futureProps: string]: any;
     }
 
-    /**
-     * @see https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview
```

Comment:
should still have this doc link somewhere, in the new file or still here on the re-export?